pkg/apis/core/validation/validation.go

```diff
@@ -914,6 +914,8 @@ func validateQuobyteVolumeSource(quobyte *core.QuobyteVolumeSource, fldPath *fie
 
 	if len(quobyte.Volume) == 0 {
 		allErrs = append(allErrs, field.Required(fldPath.Child("volume"), ""))
+	} else if strings.Contains(quobyte.Volume, "#") {
```

Comment:
I imagine there are more restrictions than this... is `'  !@$\t%^&*()<>\n"` a valid name?

Reply:
Thank you for point it out. Changed validation to be same as Quobyte UI validation.
_,-,/ and spaces are the only valid special characters for Quobyte volume (volume name can start with special character)